### ISSUE TITLE
unable to create multi-node ES cluster

### ISSUE THREAD

**Issue by jharizaj** [alexcojocaru/elasticsearch-maven-plugin#47]
I am trying to create some tests which require multiple nodes to belong to the same cluster using the `elasticsearch-maven-plugin`.

I tried setting the `instanceCount` to 2 in the pom.xml file and running `mvn verify` I am seeing two instances of ES created on port 9400 and 9401. However, when I query them for stats only a single node is reported:
```
curl -XGET http://localhost:9400/_nodes?pretty
...
  "_nodes" : {
    "total" : 1,
    "successful" : 1,
    "failed" : 0
  }
...
```

and same thing when I do `curl -XGET http://localhost:9401/_nodes?pretty`.

What I am trying to achieve is to see was `"total" : 2` for the node count in this cluster. 

I realize that the `instanceCount` option generates two separate elasticsearch instances, but is there a way to connect them so that they're part of the same cluster?  

Whatever files get created by the plugin are transient. Even changing the elasticsearch.yml files in an attempt to create a cluster gets overwritten next time `mvn verify` is run.

**Comment by alexcojocaru** [comment]
The instances should belong to the same cluster.
Here's an integration test which verifies that: https://github.com/alexcojocaru/elasticsearch-maven-plugin/blob/master/src/it/runforked-multi/src/test/java/com/github/alexcojocaru/mojo/elasticsearch/v2/MultiInstanceTest.java

**Comment by alexcojocaru** [comment]
The issue has been fixed in version 5.11. I fixed the existing multi-instance integration test to check the number of nodes in the cluster, and I have done extensive manual testing - everything looks good.

Let me know if it works for you.

**Comment by jharizaj** [comment]
Verified that it works in 5.11, thank you for the quick fix and for a great plugin!
